src/convertToObject.js

```diff
@@ -13,6 +13,24 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const cssProperties = sourceString.split(';');
+
+  const filteredCssProperies = cssProperties
+    .filter((item) => Boolean(item.trim()))
+    .map((item) => item.trim());
+
+  const cssPropertiesObject
+    = filteredCssProperies.reduce((previouslyState, property) => {
```

Comment:
Краще уникати в назвах змінних слів `object`, `array` `list` і тд
```suggestion
  const styles = filteredCssProperies.reduce(
    (previouslyState, property) => {
```